201-sql-auditing-server-policy-to-blob-storage/azuredeploy.json

```diff
@@ -46,14 +56,40 @@
       "sku": {
         "name": "Standard_LRS"
       },
-      "kind": "Storage",
-      "properties": {}
-    },
-    {
+      "kind": "StorageV2",
+      "properties": {
+        "networkAcls": {
+          "bypass": "AzureServices",
+          "virtualNetworkRules": [],
+          "ipRules": [],
+          "defaultAction": "[if(parameters('isStorageBehindVnet'), 'Deny', 'Allow')]"
+        }
+      },
+      "resources": [
+        {
+          "condition": "[parameters('isStorageBehindVnet')]",
+          "type": "Microsoft.Storage/storageAccounts/providers/roleAssignments",
+          "apiVersion": "2020-03-01-preview",
+          "name": "[concat(parameters('storageAccountName'), '/Microsoft.Authorization/', variables('uniqueRoleGuid') )]",
```

Comment:
and then this will just be the guid

Reply:
see comment about Microsoft.Authorization/roleAssignments